docs/example/manifests/split/traffic-split-50-50.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: split.smi-spec.io/v1alpha2
+kind: TrafficSplit
+metadata:
+  name: bookstore-split
+  namespace: bookstore
+spec:
+  service: bookstore.bookstore # <root-service>.<namespace>
```

Comment:
Since k8s service FQDN can be a short form as well (service, service.namespace etc.), we should clarify with a comment what are the possible allowed values.

Reply:
added a comment in traffic-split-v1.yaml and also in the doc